### PR TITLE
Extract focuses from nested `Inlined` trees when determining modification path in Scala 3 macro

### PR DESCRIPTION
**Pull request by KacperFKorban**
closes softwaremill#159

### CODE REVIEW

**Comment by adamw** [comment]
Nice! Thanks :)

File: quicklens/src/main/scala-3/com/softwaremill/quicklens/QuicklensMacros.scala [hunk 1]
```diff
@@ -326,19 +326,24 @@ object QuicklensMacros {
...
-          report.errorAndAbort(unsupportedShapeInfo(tree))
+    def extractFocus(tree: Tree): Tree = tree match {
+      /** Single inlined path */
+      case Inlined(_, _, p) =>
+        extractFocus(p)
+      /** One of paths from modifyAll */
+      case Block(List(DefDef(_, _, _, Some(p))), _) =>
+        p
+      case _ =>
+        println(tree)
```
Comment (ghostbuster91) [inline quicklens/src/main/scala-3/com/softwaremill/quicklens/QuicklensMacros.scala:337]: Leftover?

Comment (adamw) [reply quicklens/src/main/scala-3/com/softwaremill/quicklens/QuicklensMacros.scala:337]: indeed, missed that - fixing in https://github.com/softwaremill/quicklens/pull/182

Comment (adamw) [reply quicklens/src/main/scala-3/com/softwaremill/quicklens/QuicklensMacros.scala:337]: thanks :)

**Pull request merged**